skyvern/forge/sdk/artifact/manager.py

```diff
@@ -6,13 +6,21 @@
 import structlog
 
 from skyvern.forge import app
-from skyvern.forge.sdk.artifact.models import Artifact, ArtifactType
+from skyvern.forge.sdk.artifact.models import Artifact, ArtifactType, LogEntityType
 from skyvern.forge.sdk.db.id import generate_artifact_id
 from skyvern.forge.sdk.models import Step
 from skyvern.forge.sdk.schemas.observers import ObserverCruise, ObserverThought
 
 LOG = structlog.get_logger(__name__)
 
+PRIMARY_KEY = Literal[
```

Comment:
@LawyZheng we're introducing new primary keys for artifacts

Now i think we need to ensure those aio tasks for new primary keys (workflow_run_id, step_id) will be awaited at cleanup time.